pagination/pagepagination/pagination_test.go

```diff
@@ -95,6 +95,16 @@ func TestPaginationHeader(t *testing.T) {
 		assert.EqualValues(t, expect, r.Result().Header.Get("Link"))
 		assert.EqualValues(t, "5", r.Result().Header.Get("X-Total-Count"))
 	})
+
+	t.Run("Create only first if the limits provided equls the number of clients found", func(t *testing.T) {
```

Comment:
```suggestion
	t.Run("Create only first if the limits provided equals the number of clients found", func(t *testing.T) {
```

Reply:
Oh nice catch, I'll push a fix for this in a short bit after the morning meetings 👍